Generals/Code/GameEngine/Source/Common/GlobalData.cpp

```diff
@@ -983,6 +984,13 @@ GlobalData::GlobalData()
 	m_iniCRC = 0;
 	m_exeCRC = 0;
 	
+	// TheSuperHackers 12-05-2025 @skyaero
+	// To keep the VC6 version compatible with 1.08, we spoof the CRC value of the EXE
+	// This CRC is not compatible with the latest steam version (1.09).
+	// Genpatcher is required to downgrade to version 1.08.
```

Comment:
This is not accurate. The 1.08 executable can also be acquired elsewhere. I suggest to not mention unrelated tools here.